### PR TITLE
Updated syntax

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
#### Summary

Resolving remaining issues from PR #17669 @teoli2003 
#### Related issues

Resolving #16662 
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by teoli2003 — CHANGES_REQUESTED** [review]
One tiny change and we can merge this.

**Review by teoli2003 — APPROVED** [review]

File: files/en-us/web/api/network_information_api/index.md [hunk 1]
```diff
@@ -49,10 +49,10 @@ The connection object is useful for deciding whether to preload resources that t
 
 ```js
 let preloadVideo = true;
-var connection = navigator.connection || navigator.mozConnection || navigator.webkitConnection;
+const connection = navigator.connection;
 if (connection) {
   if (connection.effectiveType === 'slow-2g') {
-    preloadVideo = false;
+      preloadVideo = false;
```
Comment (Josh-Cena) [inline files/en-us/web/api/network_information_api/index.md:55]:
```suggestion
    preloadVideo = false;
```

File: files/en-us/web/api/network_information_api/index.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ The connection object is useful for deciding whether to preload resources that t
 
 ```js
 let preloadVideo = true;
-var connection = navigator.connection || navigator.mozConnection || navigator.webkitConnection;
+const connection = navigator.connection;
 if (connection) {
   if (connection.effectiveType === 'slow-2g') {
     preloadVideo = false;
```
Comment (Josh-Cena) [inline files/en-us/web/api/network_information_api/index.md:55]: This entire code block can be a one-liner `const preloadVideo = navigator.connection?.effectiveType !== 'slow-2g';` which is arguably even more readable.

Comment (teoli2003) [reply files/en-us/web/api/network_information_api/index.md:55]: Yes, I think that the example in Network Information API should be "vertically" re-read. I think there are plenty of other things to fix, like for example using `for...of`. Let's keep this for a follow-up!

**Pull request merged**